catalog/ui/catalog-ui-search/src/main/webapp/properties.js

```diff
@@ -89,9 +89,13 @@ define(function (require) {
             this.handleEditing();
             this.handleFeedback();
             this.handleExperimental();
+            this.handleListTemplates();
 
             return props;
         },
+        handleListTemplates: function() {
+            this.listTemplates = this.listTemplates.map((template) => JSON.parse(template));
```

Comment:
:pencil2: The lambda `(template) => JSON.parse(template)` can be simplified to the function reference `JSON.parse`.